protocols/timeswap/config.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "Timeswap",
+    "icon": "logo.png",
+    "metadata": {
+      "pt": [
+        {
+          "chainId": 42161,
+          "address": "0x2Be6Fab4d1408e7ad6AD91cE7B77FA2A7670782f",
+          "integrationUrl": "https://app.timeswap.io/lend/3ddff11d-768c-4b3a-8617-264be2dca259",
+          "description": "PT non-liquidatable leverage through Timeswap"
+        },
+        {
+            "chainId": 1,
+            "address": "0xAE5099C39f023C91d3dd55244CAFB36225B0850E",
+            "integrationUrl": "https://app.timeswap.io/borrow/ec9a953a-1e4c-44b7-9030-49c7596b4ee3",
+            "description": "PT non-liquidatable leverage through Timeswap"
+        },
+        {
+          "chainId": 5000,
+          "address": "0xcDA86A272531e8640cD7F1a92c01839911B90bb0",
+          "integrationUrl": "https://app.timeswap.io/borrow/f0485775-7de2-49b3-8c59-a006627715a0",
+          "description": "PT non-liquidatable leverage through Timeswap"
+        },
+        {
+            "chainId": 42161,
+            "address": "0x0c880f6761f1af8d9aa9c466984b80dab9a8c9e8",
```

Comment:
Please remove PENDLE market as we'll only show Pendle PTs/LP/YTs integration on our dApp